OrbitGl/App.cpp

```diff
@@ -123,7 +124,7 @@ void OrbitApp::OnKeyAndString(uint64_t key, std::string str) {
 }
 
 void OrbitApp::OnCallstack(CallStack callstack) {
```

Comment:
Can you rename this to something like `OnUniqueCallstack`?

Reply:
Done